packages/fxa-settings/package.json

```diff
@@ -237,7 +237,7 @@
     "grunt-contrib-watch": "^1.1.0",
     "jest-watch-typeahead": "0.6.5",
     "mutationobserver-shim": "^0.3.7",
-    "nx": "18.2.4",
+    "nx": "18.3.1",
```

Comment:
Why is nx a dep in these subpackages?

Reply:
I haven't looked into this much more since this comment from the upgrade to Nx 18 PR #16390

> Readd a few dev dependancies back into packages/**/package.json
> - Behavior of npx nx... commands seems to have changed, resulting in
> multiple "command not found: " errors, where
> refers to libaries used in packages/**/package.json scripts, such as
> tailwindcss, tsc-alias, nx, etc.